.vscode/launch.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Should we check in this file? What is it used for?

Reply:
VS Code settings for the debugger. We've checked it into other repos. Example: https://github.com/aspnet/KestrelHttpServer/tree/dev/.vscode

Reply:
It's checked into some repos, and other repos added `.vscode` to `.gitignore`.  There was no clear consensus on the engineering alias so for now this varies by repo.